jquery-news-ticker/script/news-ticker.js

```diff
@@ -0,0 +1,32 @@
+$(document).ready(function(){
+    displayNews();
+});
+
+var block_arr = $(".ticker li")
```

Comment:
Can you maintain ES6 syntax throughout the code, eg use let instead of var?

Reply:
sure!